services/movie_session.py

```diff
@@ -42,3 +44,18 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> List[dict]:
```

Comment:
In Python 3.9 and newer you can just use standard list[dict]

Reply:
Agreed